### ISSUE TITLE
require 'watir-webdriver' deprecation error

### ISSUE THREAD

**Issue by dsgn1graphics** [oesmith/puffing-billy#190]
Hi, I'm trying to configure puffing-billy with poltergeist and I'm seeing the following error. Any estimate on when this will be fixed.

```
..../puffing-billy-0.9.1/lib/billy/browsers/watir.rb:2:in `require': `require "watir-webdriver"` is deprecated. Please, use `require "watir"`
..../spec/support/capybara.rb:26:in `<top (required)>': undefined method `new=' for Billy::Browsers::Watir:Class (NoMethodError)
```

**Comment by ronwsmith** [comment]
What version of Watir are you using?  Does it work if you just update the require line in the gem as suggested?

**Comment by ronwsmith** [comment]
This will be addressed in #212
